assets/init

```diff
@@ -731,7 +731,7 @@ appStart () {
   esac
   if [ -z "${COUNT}" -o ${COUNT} -eq 0 ]; then
     echo "Setting up GitLab for firstrun. Please be patient, this could take a while..."
-    sudo -u git -H force=yes bundle exec rake gitlab:setup RAILS_ENV=production >/dev/null
+    sudo -u git -H force=yes bundle exec rake gitlab:setup RAILS_ENV=production GITLAB_ROOT_PASSWORD=${GITLAB_ROOT_PASSWORD} >/dev/null
```

Comment:
@russell I would suggest replacing `GITLAB_ROOT_PASSWORD=${GITLAB_ROOT_PASSWORD}` with `${GITLAB_ROOT_PASSWORD:+GITLAB_ROOT_PASSWORD=$GITLAB_ROOT_PASSWORD}` to avoid any breakages in the future.

`${GITLAB_ROOT_PASSWORD:+GITLAB_ROOT_PASSWORD=$GITLAB_ROOT_PASSWORD}` will make sure the `GITLAB_ROOT_PASSWORD` parameter is added to the command line only if `GITLAB_ROOT_PASSWORD` is defined.